src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
```

Comment:
You are not checking if the 'todos' prop is an array before attempting to map through it. This could lead to a crash if 'todos' is undefined. You should add a check to ensure 'todos' is an array before attempting to map over it.